Gemfile.lock

```diff
@@ -366,7 +366,7 @@ DEPENDENCIES
   mustermann (~> 0.4)
   mysql2 (~> 0.3.20)
   nifty-generators
-  paperclip (< 4.2.0)
+  paperclip (~> 4.2.2)
```

Comment:
Changes look great! But we just bumped this version -- can you revert this change? Thank you!

Reply:
done @jywarren